contrib/trace_propagators/src/main/java/io/opentelemetry/contrib/trace/propagation/B3Propagator.java

```diff
@@ -51,19 +51,31 @@
   private final B3PropagatorInjector b3PropagatorInjector;
   private final B3PropagatorExtractor b3PropagatorExtractor;
 
-  /** Creates a new instance of {@link B3Propagator}. Defaults to use multiple headers. */
-  public B3Propagator() {
-    this(false);
-  }
+  /**
+   * Creates a new instance of {@link B3Propagator}. This instance contains the reference to {@link
```

Comment:
This field won't create a new instance. The javadoc should say that it will return an instance and describe it's functionality, rather than the internal implementation.  (same comment goes for the multi-header field)

Reply:
Fixed the incorrect JavaDoc.